### ISSUE TITLE
Align keywords with second word of first line: `CALL TRANSFORMATION`

### ISSUE THREAD

**Issue by ConjuringCoffee** [SAP/abap-cleaner#22]
The rule "Align keywords with second word of first line" doesn't do anything for the command `CALL TRANSFORMATION`:

```
CALL TRANSFORMATION ztest
  SOURCE XML iv_xml
  RESULT urlrequest = rv_request.
```

Expected result:

```
CALL TRANSFORMATION ztest
     SOURCE XML iv_xml
     RESULT urlrequest = rv_request.
```

**Comment by jmgrassau** [comment]
Hi ConjuringCoffee, 

that's a good point – I did intend to exclude "CALL METHOD" from this rule (because there is a dedicated alignment rule for that), but CALL TRANSFORMATION could indeed be included. That should be easy to fix.

Kind regards,
Jörg-Michael

**Comment by ConjuringCoffee** [comment]
That's great to hear, thanks for the great work you do here!

**Comment by jmgrassau** [comment]
Hi ConjuringCoffee, 

this, too, should now be resolved in the new release (version 1.2.1)!

Kind regards,
Jörg-Michael

**Comment by ConjuringCoffee** [comment]
I can confirm that it works - thanks!
